src/Network/Network/PrivateLinkService/PrivateLinkServiceIpConfiguration/NewAzurePrivateLinkServiceIpConfiguration.cs

```diff
@@ -75,12 +84,7 @@ public override void Execute()
             {
                 ipconfig.Subnet = this.Subnet;
             }
-
-            if(PublicIpAddress != null)
```

Comment:
So, this is actually a breaking behavior change, how is this not cause existing scripts to fail?

Reply:
There should't be any script outside of Azure that is using the this feature since it's been protected by feature flag on Subscription Level. Besides, if there is a script for it, the property is totally redundant and never been used from Server side (NRP). So it won't casue any fail.